quill-engine/src/main/scala/io/getquill/norm/ApplyMap.scala

```diff
@@ -8,9 +8,9 @@ import io.getquill.sql.Common.ContainsImpurities
 
 /**
  * Notes for the conceptual examples below. Gin and Tonic were used as prototypical
- * examples of things that "are joined". In the table form, they are alude to the following
+ * examples of things that "are joined". In the table form, they are aliased to the following
  * tonics is Query[Tonic], tonic is Tonic
- * gins is Query[Gin], is Gin
+ * gins is Query[Gin], gin is Gin
```

Comment:
I think this is the only "in the neighborhood" fix. (Someone else had opinions about `Water` below, but that's too complicated for me.)